src/status_im2/subs/wallet_2/wallet.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im2.subs.wallet-2.wallet
+  (:require [re-frame.core :as re-frame]
+            [clojure.string :as string]))
+
+(defn calculate-raw-balance
+  [raw-balance decimals]
+  (let [raw-number (js/parseInt raw-balance)]
```

Comment:
@mmilad75, there's a better function to parse integers. Take a look at `utils.number/parse-int`.

Notice that we can explicitly say a default value, which is great when combined with `if-let` if we default to `nil`. By default, as in many programming languages, `utils.number/parse-int` defaults to 0 if the second argument is not passed.

```clojure
(defn calculate-raw-balance
  [raw-balance decimals]
  (if-let [n (utils.number/parse-int raw-balance nil)]
    (/ n (Math/pow 10 (utils.number/parse-int decimals)))
    0))
```

Reply:
Great suggestion! Done